### ISSUE TITLE
Sometimes NVDA Version 2025.1beta3 (2025.1.0.36199) Remains Silent When Selecting the Text in Microsoft word or in the .docx written file

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#18043]
### Steps to reproduce:
Press Windows key to activate search box;
press Tab key to reach at the navigation menu list where folders are pinned;
Press right arrow to reach at the document folder;
Press Enter to open it;
down arrow to find a respective .docx file, mine is called: Chawala draft 3 Summarized chapter four 60 Page Version.docx - Word
Select the text by pressing shift+any arrow key or control+shift+any arrow key; it sometimes remains silent. 
### Actual behavior:

Sometimes, when pressing text selection keys, shift+any arrow key or control+shift+any arrow key on the text you want to select, NVDA remains silent, it does not announce what is selected
### Expected behavior:

when going on pressing text selection keys, shift+any arrow key or control+shift+any arrow key on the text you want to select, NVDA should always announce the selected text and say "selected" at last
#### NVDA installed/portable/running from source:
NVDA Installer
#### NVDA version:
NVDA Version 2025.1beta3 (2025.1.0.36199)
#### Windows version:
Windows 11 Pro 64-bit (10.0, Build 26100) (26100.ge_release.240331-1435)
#### Name and version of other software in use when reproducing the issue:
Microsoft® Word for Microsoft 365 MSO (Version 2503 Build 16.0.18623.20208) 64-bit 
#### Other information about your system:
Language: English (Regional Setting: English)
System Manufacturer: HP
System Model: HP ENVY x360 Convertible 15-cn0xxx
BIOS: F.30 (type: UEFI)
Processor: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 CPUs), ~2.0GHz
Memory: 16384MB RAM
Available OS Memory: 16204MB RAM
Page File: 4973MB used, 12255MB available
Windows Dir: C:\WINDOWS
DirectX Version: DirectX 12

#### Does the issue still occur after restarting your computer?
Yes, when NVDA remains silent when selecting the text, i restart NVDA itself.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes, the same behaviour
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

[nvda.log](https://github.com/user-attachments/files/20009619/nvda.log)

**Comment by seanbudd** [comment]
Which other versions of NVDA did you test? Do you mind if I remove the version number from the title? It implies this is an issue introduced in 2025.1

**Comment by jacobchawala** [comment]
yes i tested nvda_2024.4.2. the number you want remove on my title, i took it in the about section in the NVDA help menu.

**Comment by Adriani90** [comment]
@jacobchawala your MS office version seems to be outdated. Can you please update to MS office 365 build 2505 last update? Are you then having this issue with NVDA 2025.1 RC1?
I cannot really reproduce this issue.
